reports/Washington.md

```diff
@@ -34,7 +34,21 @@ id: wa-seattle-1
 **Links**
 
 * https://mobile.twitter.com/chadloder/status/1267011092045115392
-* 2020OPA-0324 - May 30 - "Placing the knee on the neck area of two people who had been arrested" (possibly)
+* Possibly related to OPD Case 2020OPA-0324 - "Placing the knee on the neck area of two people who had been arrested"
+
+
+### Flashbang causes serious thumb and forefinger injury | May 30th
+
+According to a news report, a oman at the Saturday, June 1st protest in downtown Seattle sustained severe injuries from sharpnel that pierced her thumb and forefinger.
+
+tags: flashbang
+
+id: wa-seattle-55
+
+**Links**
+
+* https://www.seattletimes.com/seattle-news/seattle-police-continue-to-use-flash-bang-grenades-during-protests-despite-recommendations/
```

Comment:
tbh I don't think this is solid enough to call an incident. One person recalling that she saw an incident and telling a reporter later is not quite concrete.